Adding RSVP Keeper to the Apps/Websites section

johndatserakis opened the pull request:
Adding RSVP Keeper to the Apps/Websites section

RSVP Keeper is a full-featured online reservation app built with Vue and Go.

https://www.rsvpkeeper.com/

johndatserakis commented:
Hey @Scrum, I just merge in the latest from master. Everything's merged in and ready to go.

Thank you.

Scrum commented on README.md:
```diff
@@ -976,9 +976,10 @@
   - [Translator-vuejs](https://manuel-suarez-abascal.github.io/translator-vuejs/) - Translation App built with Vuejs, Yandex API & ResponsiveVoice.js API.
   - [Big Timer](https://www.bigtimer.net/) - Fullscreen countdown timer for workshops, meetings and presentations. Big Timer helps workshop facilitators, meeting chairs, design sprinters, presenters and aspiring game show hosts stick to their program.
   - [Wirenook](https://wirenook.com/) - Free online app for building responsive website wireframes. Hi- and Lo- Fidelity, project sharing and svg download.
+  - [RSVP Keeper](https://www.rsvpkeeper.com/) - Online reservations made easy. Get your event up and running in no time. Made with Vue and Go.
```
@johndatserakis Needs to be at the end of the list.

Scrum commented on README.md:
```diff
@@ -976,9 +976,10 @@
   - [Kvalitetskontroll](https://www.kvalitetskontroll.no/) - Norwegian management system tailored for the construction industry.
   - [Poolside FM](https://poolside.fm/) - A retro-style music player
-  - [Inoreader.com](https://inoreader.com/) - One of the biggest RSS readers and news aggregators out there. 
+  - [Inoreader.com](https://inoreader.com/) - One of the biggest RSS readers and news aggregators out there.
```
@johndatserakis Needs to be at the end of the list.

Scrum commented on README.md:
```diff
@@ -1411,7 +1412,7 @@ Tooltips / popovers
  - [vue-video-section](https://github.com/johndatserakis/vue-video-section) - A simple video header/section component for Vue. Good for video backgrounds and overlaying content on them.
  - [vue-waveform](https://github.com/chenqiaoen521/vue-waveform) -It is use canvas waveform audio player wavesurfer component  contains timeline bar line for Vue 2.x
- - [vue-responsive-video-background-player](https://github.com/avidofood/vue-responsive-video-background-player) - Play your own videos in background responsively in different resolutions. 
+ - [vue-responsive-video-background-player](https://github.com/avidofood/vue-responsive-video-background-player) - Play your own videos in background responsively in different resolutions.
```
@johndatserakis Needs to be at the end of the list.

johndatserakis commented:
Hey @Scrum, sorry for the confusion.

johndatserakis commented:
Hey there @Scrum,

Sorry for the confusion. The other lines were not added by me and are not my projects, it seems like they just got mixed in on my recent merge.

So to make sure everything's nice and clean, I removed all my previous changes, merged in a fresh copy of master, and added back my entry so there's a clean diff with only my single line change. It should be ok now.

Sorry and thank you.
